package.json

```diff
@@ -62,10 +62,12 @@
   "main": "built/index.js",
   "typings": "built/index.d.ts",
   "scripts": {
+    "postinstall": "cd bin && webdriver-manager update && npm run install_website && npm run install_testapp",
```

Comment:
Adding a `postinstall` like this will cause anyone who used `npm install protractor` to install the testapp - something almost no one wants and won't even work since the testapp is ignored via `.npmignore`.